newrelic/resource_newrelic_entity_tags.go

```diff
@@ -161,10 +165,6 @@ func resourceNewRelicEntityTagsUpdate(ctx context.Context, d *schema.ResourceDat
 
 		currentTags := convertTagTypes(t)
 
-		if err != nil {
-			return resource.NonRetryableError(fmt.Errorf("error retrieving entity tags for guid %s: %s", d.Id(), err))
-		}
-
```

Comment:
@shashank-reddy-nr why do we need to remove this? (if we have discussed this already, can you help me recall please 🤔)

Reply:
oh yeah 🤦 right 👍